packages/fluentui/e2e/cypress.json

```diff
@@ -1 +1,7 @@
-{ "video": false, "screenshotOnRunFailure": false, "integrationFolder": "./tests", "testFiles": "**.spec.ts" }
+{
+  "ignoreTestFiles": "*?ialog*.spec.ts",
```

Comment:
I would rather use `x` prefix inside tests (for example, `xdescribe`), but this also works.